events/imperia_infamy_rebalance_events.txt

```diff
@@ -548,13 +548,12 @@ infamy_rebalance.10 = {
 			value = 1.3
 		}
 		infamy >= pariah_infamy_SCOPE_VALUE_MULT_VAR_trigger
-		NOT = {
+		NOR = {
 			has_modifier = modifier_pariah_economic_sanctions
 			has_modifier = modifier_pariah_international_isolation
 		}
 	}
-	option = {
-		name = infamy_rebalance.10.a
+	immediate = {
```

Comment:
Any idea why this prevents the firing?

Reply:
Not sure what you mean, but I'll try to answer what I think you mean :p 
The modifier is now in the immediate section, whereas before it was part of the option, so if you didnt click anything it *could* pop multiple times.
I also changed a NOT to a NOR, since that made a lot more sense to me. Would probably pop a lot before that, as well..